blazor/3D-chart/appearance.md

```diff
@@ -0,0 +1,559 @@
+---
+layout: post
+title: Appearance in Blazor 3D Chart Component | Syncfusion
+description: Checkout and learn here all about Appearance Customization in Syncfusion Blazor 3D Chart component and much more.
+platform: Blazor
+control: 3D Chart
+documentation: ug
+---
+
+# Appearance in Blazor 3D Chart Component
+
+## Custom color palette
+
+The default color of series or points can be customized by providing a custom color palette of your choice by using the [Palettes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.SfChart.html#Syncfusion_Blazor_Charts_SfChart_Palettes) property.
+
+```cshtml
+
+@using Syncfusion.Blazor.Chart3D
+
+<SfChart3D Title="Olympic Medals" Palettes="@palettes" WallColor="transparent" EnableRotation="true" RotationAngle="7" TiltAngle="10" Depth="100">
+    <Chart3DPrimaryXAxis ValueType="Syncfusion.Blazor.Charts.ValueType.Category">
+    </Chart3DPrimaryXAxis>
+
+    <Chart3DSeriesCollection>
+        <Chart3DSeries DataSource="@MedalDetails" XName="Country" YName="Gold" Type="Chart3DSeriesType.Column">
+        </Chart3DSeries>
+        <Chart3DSeries DataSource="@MedalDetails" XName="Country" YName="Silver" Type="Chart3DSeriesType.Column">
+        </Chart3DSeries>
+        <Chart3DSeries DataSource="@MedalDetails" XName="Country" YName="Bronze" Type="Chart3DSeriesType.Column">
+        </Chart3DSeries>
+    </Chart3DSeriesCollection>
+</SfChart3D>
+
+@code{
+    public class Chart3DData
+    {
+        public string Country { get; set; }
+        public double Gold { get; set; }
+        public double Silver { get; set; }
+        public double Bronze { get; set; }
+    }
+
+    public List<Chart3DData> MedalDetails = new List<Chart3DData>
```

Comment:
@IlamathiSF3875 
Data alignments is not proper, can you format the code and add?